package/resources/logback.xml

```diff
@@ -0,0 +1,29 @@
+<configuration>
+
+    <property name="basePath" value="${sys:catalina.base}" />
+
+    <appender name="file" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>${basePath}/logs/openerp-feed-service.log</file>
+        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <fileNamePattern>${basePath}/logs/openerp-feed-service-%d{MM-dd-yyyy}-%i.log</fileNamePattern>
+            <maxFileSize>5000KB</maxFileSize>
+            <maxHistory>10</maxHistory>
+        </rollingPolicy>
+        <encoder>
+            <pattern>%d %-5p [openerp-atomfeed-service][%c] %m%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="console" class="ch.qos.logback.core.ConsoleAppender">
+        <target>SYSTEM_OUT</target>
+        <encoder>
+            <pattern>%d %-5p [openerp-atomfeed-service][%c] %m%n</pattern>
+        </encoder>
+    </appender>
+
+    <root level="INFO">
```

Comment:
Debug level maybe until Odoo integration is under development?

Reply:
Done